sql/mob_droplist.sql

```diff
@@ -21915,12 +21915,16 @@ INSERT INTO `mob_droplist` VALUES (2639,0,0,1000,1031,30); -- Gusgen Chest Key (
 INSERT INTO `mob_droplist` VALUES (2639,2,0,1000,880,0);   -- Bone Chip (Steal)
 
 -- ZoneID: 192 - Wendigo Blm
--- ZoneID: 198 - Wendigo Blm
-INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,880,210);    -- Bone Chip (21.0%)
-INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,544,90);     -- Pile Of Answer Sheets (9.0%)
+INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,880,@VCOMMON);    -- Bone Chip (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,544,@RARE);     -- Pile Of Answer Sheets (Rare, 5%)
 INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,4824,@RARE); -- Scroll Of Gravity (Rare, 5%)
 INSERT INTO `mob_droplist` VALUES (2640,2,0,1000,880,0);      -- Bone Chip (Steal)
 
+-- ZoneID: 198 - Wendigo Blm
+INSERT INTO `mob_droplist` VALUES (3238,0,0,1000,880,@VCOMMON);    -- Bone Chip (Very Common, 24%)
```

Comment:
ID 3238 is already in use. Can you relocate this drop group to the bottom of the stack and pick the next available ID instead?

Reply:
![imagen](https://user-images.githubusercontent.com/60053999/225323139-f799d63b-9af0-4b63-a8b3-0eb1d5829bb4.png)

Reply:
Ah, sorry gents. Corrected.